sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -13,10 +11,12 @@
 
 ### Bugs Fixed
 
-- Use the package version for telemetry, rather than api version.
+- Use the package version for telemetry, rather than API version.

```

Comment:
@gearama, I hope you don't mind - I could not propose this as a suggestion commit comment as normal, because the line is behind some range of recently changed lines, so I committed this change to your branch directly.

Reply:
Thankl you for your help. :D